packages/experimental/Shadow/src/shadowStyle.android.ts

```diff
@@ -0,0 +1,25 @@
+import { memoize } from '@fluentui-react-native/framework';
+import { ShadowToken } from '@fluentui-react-native/theme-types';
+
+/**
+ * For Android, we do not set two independent shadows on two views.
+ * Instead, the `elevation` prop takes care of synthesizing a key and ambient
+ * shadow together with it's own depth ramp. The `blur` token of our key shadow roughly
+ * matches the elevation value. See the following:
+ * https://material.io/design/environment/light-shadows.html#light
+ * https://github.com/microsoft/apple-ux-guide/blob/gh-pages/Shadows.md
```

Comment:
Just fyi the screenshots I have for Android in https://github.com/microsoft/apple-ux-guide/blob/gh-pages/Shadows.md look pretty different from the screenshots for iOS, so referencing that doc might be confusing